docker-cloud/cloud-swarm/link-azure-swarm.md

```diff
@@ -0,0 +1,87 @@
+---
+previewflag: cloud-swarm
+description: Link your Microsoft Azure account
+keywords: Azure, Cloud, link
+title: Link to Microsoft Azure Cloud Services to create Swarms
+---
+
+You can link your Microsoft Azure account so that Docker Cloud can provision and
+manage swarms on your behalf. For this, you will need an SSH key and your Azure
+subscription ID to authenticate Docker to your Cloud provider.
+
+## Setting up SSH keys
+
+{% include content/ssh-overview.md %}
+
+### Check for existing SSH keys
+
+You may not need to generate a new SSH key if you have an existing key that you
+want to multi-purpose.
+
+{% include content/ssh-find-keys.md %}
+
+If you find an existing key you want to use, skip to the topic that describes
+how to [copy your public key for use with Docker
+Cloud](copy-your-public-key-for-use-with-docker-cloud).
+
+Otherwise, [create a new SSH key](create-a-new-ssh-key-for-use-by-docker-cloud).
+
+### Create a new SSH key
+
+{% include content/ssh-gen-keys.md %}
```

Comment:
It occurs to me that having `_includes/content/` doesn't really provide much organizational benefit. Maybe make a folder called `_includes/ssh` and use that for these topics? We probably need to come up with a sensible way to organize this stuff as we try to leverage reuse more.